src/reactpy/types.py

```diff
@@ -871,6 +880,7 @@ class VdomJson(TypedDict):
     children: NotRequired[list[Any]]
     attributes: NotRequired[VdomAttributes]
     eventHandlers: NotRequired[dict[str, JsonEventTarget]]
+    jsExecutables: NotRequired[dict[str, JavaScript]]
```

Comment:
I'm not the biggest fan of the `jsExecutables` nomenclature, since it doesn't seem to make it "blindingly obvious" to the uneducated reader as to what this is.

Maybe `jsEventHandlers` is the more appropriate name? Then, we can rename the existing `eventHandlers` to `pythonEventHandlers`?

Reply:
That was actually my exact initial leaning. I only hesitated because 1) I didn't want to necessarily change anything existing and 2) I suppose I don't fully like `eventHandler` for this javascript concept, again, because of my main use case where it's not being executed in response to an event at all, but simply as configuration. But I could be persuaded easily, since it was mainly my first reason that swayed me. If you're good with that, I can make the change. I even thought `pyEventHandlers` and `jsEventHandlers`, to match them up a bit more. Final thoughts?

Reply:
I agree that `eventHandlers` isn't the right name either... Need to brainstorm some better names.

Perhaps `inlineJavascript`?

Reply:
Hm... Yeah, I like `inlineJavascript`. I can go ahead and make that change. If we come up with something better later, then I don't see changing it being a big deal for users. Most of the Vdom stuff is black-boxed to the user.

Reply:
Done.